CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 This file is used to list changes made in each version of the devopsbox cookbook.
 
+## Master
+ * install ncdu package.
+
```

Comment:
Not yet, as we don't publish this cookbook to supermarket. BTW are we going to ?